services/authentication/app.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+from flask import Flask, request, jsonify, redirect
+
+from db import Base, engine
+from resources.login import Login
+from resources.roles_updates import RolesUpdates
+from resources.roles import Roles
+
+app = Flask(__name__)
+app.config["DEBUG"] = True
+Base.metadata.create_all(engine)
+
+@app.route('/login/<uuid>/<digid_message>', methods=['GET'])
+def verify_login(uuid, digid_message):
+    try:
+        uuid = int(uuid)
+    except ValueError:
+        return jsonify({'message': 'The id must be an integer'}), 500
+    return Login.digid_check(uuid, digid_message)
+
+@app.route('/login/<uuid>/service_choice', methods=['GET'])
```

Comment:
So after verifying the login, you make the user login again to retrieve their services via a call? Why not immediately after the first login?

Reply:
I simulate a login via DigID. If the login from DigID is accepted, users are automatically redirected to the service_choice (see `login.py`). If the login from DigID is rejected, again simulation, users get a message that they have to try again.